layers/+os/osx/keybindings.el

```diff
@@ -41,24 +41,30 @@
 
   ;; Set internal variables according to the given config variables
   (cl-loop for (key-var . internal-var) in modifier-keys do
-           (let ((key-value (symbol-value key-var)))
-             (when (member key-value allowed-values)
-               (setf (symbol-value internal-var) key-value))))
+    (let ((key-value (symbol-value key-var)))
+      (when (member key-value allowed-values)
+        (setf (symbol-value internal-var) key-value))))
 
   (defun kbd-mac-command (keys)
     "Wraps `kbd' function with Mac OSX compatible Command-key (⌘).
+     (when osx-swap-option-and-command
```

Comment:
These two lines have no effect, because they are in the `kbd-mac-command` functions docstring.

Before your last force push: https://github.com/syl20bnr/spacemacs/commit/61d0d97bddf37608dd10a38fd84370ffb28b3cc6
you had these two lines above the function definition for: `  (defun kbd-mac-command (keys)`

That's probably where they should be placed, then they only will run once when the `osx/keybindings.el` file is evaluated.

Leave an empty line between them and the start of the following function definition.